web/src/components/CardForm.js

```diff
@@ -49,12 +58,19 @@ const CardForm = ({
                     )}
                   </Button>
                 )}
-                <Button variation="transparent" onClick={history.goBack}>
-                  Cancel
-                </Button>
+                {cancelable && (
+                  <Button variation="transparent" onClick={history.goBack}>
+                    Cancel
+                  </Button>
+                )}
               </div>
             </fieldset>
           </form>
+          {footer && (
+            <div className="ds-u-margin-top--4 ds-u-border-top--1 ds-u-color--muted">
```

Comment:
Wondering if this should be a `<footer>` styled as a child of `.card--container` instead of using atomic styles here. 🤔